docs/reference/commandline/create.md

```diff
@@ -68,7 +68,11 @@ Creates a new container.
       -u, --user=""                 Username or UID
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
-      -v, --volume=[]               Bind mount a volume
+      -v, --volume=[]               Bind mount a volume with: [host-dir:]container-dir[:<options>], where
```

Comment:
perhaps we should mention `[host-dir | volume-name:]`, because we now support named volumes

(these options are hard to document though, without becoming confusing :cry:)